processes/host-selection/host-prioritization.md

```diff
@@ -0,0 +1,37 @@
+# Meeting to decide host prioritization
+
+**DATE:** 2019-23-01
+
+**PARTICIPANTS:** @Jethet, @eskape16, @daverick, @amaliacardenas
+**LOCATION:** Cloud Co-working
+
+## TL;DR
+We want to be more transparent with the way we organize events. In this way anyone who attends or coaches at codebar knows our needs and if interested can help us out. The reason why we want to do this is to have more members from the community participate in the organization of events and to help us as we expand.
+
+In this document, you will find
+1. The criteria that we use when we prioritize where to host codebar events.
+2. A guide for how to contact hosts who want to sponsor a workshop.
+
+# Host prioritization criteria
+
+## Must Have
+* **Aligns with the codebar mission**
+* **Safe place**
+* **Food and drinks**
+* **Comfortable space for workshops**
+
+## Important
+* **Accessibility within the city**
+* **Maximum capacity**
+
+## Nice to have
+* **Comfortable space for talks**
```

Comment:
We have considered the space that is needed at the start of the workshop, for the introductions (Codebar & host organisation and doing the round of people introducing themselves. The current set-up of standing in a circle is ideal in my opinion: it's an informal setting for seeing everyone, having eye contact. 
Some hosts do not have a space that is large enough for this, however. Although we have put this under 'Nice to have', it is the start of the workshop and is essential to get a relaxed and welcoming atmosphere that is such a major element of feeling safe.
There may be other options (e.g. using a meeting room like we do when we are at Qlik in the Blue Building or a room that is used for trainings) but ideally, we want an open space large enough to have 30 people standing in a circle.
Suggestion: this could be something to discuss with the host during a visit.
Do we go to a host location beforehand to have a look? 
Do we want to set a minimum requirement of floor space? 
Do we discuss with them about moving furniture etc.? 
Can we have a team of volunteers who are willing to come earlier and help get a place ready (and help get things back in place afterwards)?

Reply:
> Can we have a team of volunteers who are willing to come earlier and help get a place ready (and help get things back in place afterwards)?
We the organizer are in a way already doing this at some location, were we arrive before the official start of the workshop to help the host and we usually left the last to be sure to let everything clean and tidy.